Directory.Build.props

```diff
@@ -4,15 +4,14 @@
   </PropertyGroup>
 
   <PropertyGroup Label="Target Platforms" >
-    <DefaultNetCoreVersion>net7.0</DefaultNetCoreVersion>
-    <NetCoreVersions>$(DefaultNetCoreVersion);net6.0</NetCoreVersions>
+    <NetCoreVersions>net7.0</NetCoreVersions>
```

Comment:
Just to clarify, I'm considering this as working normally, having that with only one .NET Core version the default one should not be necessary.